source/site-administrators/engine/engine-saml2-configuration.rst

```diff
@@ -0,0 +1,126 @@
+|``role.mappings.mapping``          |List of mappings to apply for roles, every |                                     |
+|                                   |role sent by the IDP will be compared      |                                     |
+|                                   |against this list                          |                                     |
++-----------------------------------+-------------------------------------------+-------------------------------------+
+|``keystore.defaultCredential``     |The name of the default credential to use  |                                     |
++-----------------------------------+-------------------------------------------+-------------------------------------+
+|``keystore.path``                  |The path of the keystore file in the repo  |``/config/engine/saml2/keystore.jks``|
++-----------------------------------+-------------------------------------------+-------------------------------------+
+|``keystore.password``              |The password of the keystore file          |                                     |
++-----------------------------------+-------------------------------------------+-------------------------------------+
+|``keystore.credentials.credential``|List of credentials in the keystore        |                                     |
++-----------------------------------+-------------------------------------------+-------------------------------------+
+|``identityProviderName``           |The name of the identity provider to use   |                                     |
++-----------------------------------+-------------------------------------------+-------------------------------------+
+|``identityProviderDescriptor``     |The path of the identity provider metadata |``/config/engine/saml2/idp.xml``     |
+|                                   |XML descriptor in the repo                 |                                     |
++-----------------------------------+-------------------------------------------+-------------------------------------+
+|``serviceProviderName``            |The name of the service provider to use    |                                     |
++-----------------------------------+-------------------------------------------+-------------------------------------+
+|``serviceProviderDescriptor``      |The path of the service provider metadata  |``/config/engine/saml2/sp.xml``      |
+|                                   |XML descriptor in the repo                 |                                     |
++-----------------------------------+-------------------------------------------+-------------------------------------+
+
+-------------------------
+Commit the required files
+-------------------------
+
+You will need to add & commit the keystore and descriptor files manually to the site repository, the location will
+depend of the configuration used, the following example uses the default locations:
```

Comment:
Typo? use `on` instead of `of`  and break the sentence like so...
```
the location will depend on the configuration used.  The following example uses the default locations:
```